File: lab7/TestFestival.java
public class TestFestival {
	public static void main(String[] args) {
		Act myAct1 = new Act(1, "Russel Howard", "Comedian", "O2 Arena");
		Act myAct2 = new Act(10, "Dancing Team", "Dance", "O2 Academy Brixton");
		Act myAct3 = new Act(1, "The Eden Project", "Music", "02 Academy Shepherds Bush");

		LineUp myFestival = new LineUp();

		myFestival.addAct(myAct1);
		myFestival.addAct(myAct2);
		myFestival.addAct(myAct3);

		System.out.println(myFestival);
		int i = 0;

		while(i < 28) {
			LineUp.addAct(myAct1);
			i++;
		}

		System.out.println(myFestival.numberOfActs());
		System.out.println(myFestival.totalMembers());
	}
}

File: lab7/test_festival.py
import festival

myAct1 = festival.Act(1, "Russel Howard", "Comedian", "O2 Arean")
myAct2 = festival.Act(10, "Dancing Team", "Dance", "O2 Academy Brixton")
myAct3 = festival.Act(1, "The Eden Project", "Music", None)

myFestival = festival.LineUp()

myFestival.add_act(myAct1)
myFestival.add_act(myAct2)
myFestival.add_act(myAct3)

print(myFestival)

i = 0

#while i < 28:
#    myFestival.add_act(myAct1)
#    i = i + 1

myFestival.numberOfActs()
myFestival.totalMembers()
myFestival.totalKinds()
